libs/database/src/lib/entities/disturbance.entity.ts

```diff
@@ -40,14 +40,6 @@ export class Disturbance extends Model<Disturbance> {
   @Column(TEXT)
   description: string | null;
 
-  @AllowNull
-  @Column(INTEGER.UNSIGNED)
-  oldId: number;
-
-  @AllowNull
-  @Column(STRING)
-  oldModel: string | null;
```

Comment:
In cases like this, I think we want the Entity to still match what's in the DB. I've been adding a `@deprecated` flag with a comment to indicate that we can likely drop the column later. Lots of examples in [Media](https://github.com/wri/terramatch-microservices/blob/03865417d32b59fc741b62e3a4a46bddb8866374/libs/database/src/lib/entities/media.entity.ts#L120)